src/main/webapp/scripts/sessioncache.js

```diff
@@ -15,16 +17,20 @@ class SessionCache {
    *    session the client is in, holds information such as the
    *    session ID and the screen name of the current user.
    * @param {number=} [refreshCadence = 30000] Represents the cadence at
-   *    which the sessionInformation is refreshed. By default, the rate is 
+   *    which the sessionInformation is refreshed. By default, the rate is
    *    30,000 milliseconds (or 30 seconds).  
+   * @param {number=} [pollingCadence = 30000] Represents the 
+   *    cadence at which polling occurs. By default, the rate is 
+   *    30,000 milliseconds (or 30 seconds). 
    */
-  constructor(urlParams, refreshCadence = 30000) {
+  constructor(urlParams, refreshCadence = 30000, pollingCadence = 30000) {
```

Comment:
Whats the difference between refreshCadence and pollingCadence? You poll at the refresh cadence. Not sure why there's another cadence.